performance-tests/bench_2/worker/configs/simple_action_content_filtered_topics_config.json

```diff
@@ -0,0 +1,111 @@
+{
+  "enable_time": { "sec": -1, "nsec": 0 },
+  "start_time": { "sec": -3, "nsec": 0 },
+  "stop_time": { "sec": -10, "nsec": 0 },
+  "destruction_time": { "sec": -1, "nsec": 0 },
+
+  "process": {
+    "config_sections": [
+      { "name": "common",
+        "properties": [
+          { "name": "DCPSSecurity",
+            "value": "0"
+          },
+          { "name": "DCPSDebugLevel",
+            "value": "0"
+          }
+        ]
+      }
+    ],
+    "discoveries": [
+      { "name": "bench_test_rtps",
+        "type": "rtps",
+        "domain": 7
+      }
+    ],
+    "instances": [
+      { "name": "rtps_instance_01",
+        "type": "rtps_udp",
+        "domain": 7
+      }
+    ],
+    "participants": [
+      { "name": "participant_01",
+        "domain": 7,
+        "transport_config_name": "rtps_instance_01",
+
+        "qos": { "entity_factory": { "autoenable_created_entities": false } },
+        "qos_mask": { "entity_factory": { "has_autoenable_created_entities": false } },
+
+        "topics": [
+          { "name": "topic_01",
+            "type_name": "Bench::Data",
+            "content_filtered_topics": [
+              {
+                "cft_name": "cft_1",
+                "cft_expression": "msg_count > %0",
```

Comment:
Looking at the Bench::Data IDL structure, it doesn't look like there's that much we can usefully filter against. Let's add a field to it called filter_class or msg_class or something (ulong is fine to keep the size down) and then update the WriteAction to populate it. We may need to have a little bit of logic in there for usefully being able to iterate over over "acceptable" values for it (similar to what I described for the CFT action, but not as complex).